spec/presenters/navigation_presenter_spec.rb

```diff
@@ -11,6 +11,7 @@
         let(:user) { build(:user, personal_key: 'abcdef') }
```

Comment:
It would be good to add a context for "for a user with deactivated profile", using the profile factory's `:password_reset` trait.

https://github.com/18F/identity-idp/blob/fe11234a14ba5426212e37ff0106f35c5f14711a/spec/factories/profiles.rb#L18-L22

Reply:
Added in https://github.com/18F/identity-idp/pull/6452/commits/b72588d47cadd24fd0aa7aad6315d52c2237a075